src/main/java/gregicadditions/recipes/categories/circuits/CircuitRecipes.java

```diff
@@ -984,12 +984,80 @@ private static void opticalCircuits() {
 
     private static void exoticCircuits() {
 
-        /* COMING SOON...
-
         for (FluidStack fluidStack : SOLDER_FLUIDS) {
             fluidStack.amount = Math.min(64000, fluidStack.amount * 4);
```

Comment:
```suggestion
            FluidStack fluidStackx4 = new FluidStack(fluidStack.getFluid(), Math.min(64000, fluidStack.amount * 4));
```
This is necessary for the soldering fluid bug to not return, where soldering fluids take insane amounts of fluid due to updating a pointer. Make sure to update the use below at line 998.